### ISSUE TITLE
doc needed for multiple zpools

### ISSUE THREAD

**Issue by shivno** [iocage/libiocage#236]
hi again,

not using libiocage yet, but i guess the work is done here.
currently multiple zpools are not supported and i dont know about other setups, but for me every 2nd server has them, due to historic reasons, hdd/sdd or else.

perhaps a doc for other people and/or best practises in this case would be helpful. perhaps i could write this later.

**Comment by gronke** [comment]
Hi @shivno! Multiple zpools are possible if you import the library in Python and script the feature. In first place I'd like to implement the missing features from other iocage versions before focussing on future features. The interfaces you find in the lib to achieve running jails from multiple zpools are unlikely to change btw.

We can define a new Feature that implements multi zpool support. IMO this should be implemented in the lib and CLI, so that both list jails from all active pools. The activate/deactivate commands would in this case even become more important.

**Comment by gronke** [comment]
The multi data source support feature is discussed in https://github.com/iocage/libiocage/issues/285. When it is implemented, there should also be proper user documentation available.

**Comment by gronke** [comment]
This was merged with #287. Instead of just supporting multiple ZFS pools we now support multiple source datasets.

```
sysrc ioc_dataset_one="zroot/iocage"
sysrc ioc_dataset_two="zroot/other-dataset/iocage"
```
